src/components/page-details.jsx

```diff
@@ -43,18 +43,24 @@ export default class PageDetails extends React.Component {
 
   componentWillUnmount () {
     window.removeEventListener('keydown', this);
+    this.setTitle(true);
   }
 
   /**
    * @param {PageDetailsProps} previousProps
    */
   componentDidUpdate (previousProps) {
+    this.setTitle();
     const nextPageId = this.props.match.params.pageId;
     if (nextPageId !== previousProps.match.params.pageId) {
       this._loadPage(nextPageId);
     }
   }
 
+  setTitle (unmounting=false) {
```

Comment:
Sorry to be _that_ person but could you add whitespace for `unmounting=false` if it's not too much trouble? Not a big deal if not. Otherwise looks great to me!

Reply:
No worries, was easy enough to go in and fix!  Just pushed the latest commit!